cli/providers/aws/ec2/securitygroups.go

```diff
@@ -0,0 +1,95 @@
+package ec2
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/pchico83/i2kit/cli/schemas/environment"
+)
+
+//CreateSG creates the project security group
+func CreateSG(e *environment.Environment, config *aws.Config) error {
+	vpc, err := GetVPC(e, config)
+	if err != nil {
+		return err
+	}
+	svc := ec2.New(session.New(), config)
+	description := fmt.Sprintf("Security Group for environment %s", e.Name)
+	name := fmt.Sprintf("%s-i2kit", e.Name)
+	csgi := &ec2.CreateSecurityGroupInput{
+		Description: &description,
+		GroupName:   &name,
+		VpcId:       &vpc,
+	}
+	sg, err := svc.CreateSecurityGroup(csgi)
+	if err != nil {
+		if !strings.Contains(err.Error(), "InvalidGroup.Duplicate") {
+			return err
+		}
+		dsgi := &ec2.DescribeSecurityGroupsInput{
+			GroupNames: []*string{&name},
+		}
+		sgs, err2 := svc.DescribeSecurityGroups(dsgi)
+		if err2 != nil {
+			return err2
+		}
+		for _, i := range sgs.SecurityGroups {
```

Comment:
do we need to handle the case where DescribeSecurityGroups doesn't return the group? (e.g. if there's a group with the same name, but on a different vpc)

Reply:
No, it wouldn't fail on that case